install-panel.sh

```diff
@@ -84,6 +84,12 @@ CONFIGURE_FIREWALL_CMD=false
 # firewall status
 CONFIGURE_FIREWALL=false
 
+# build panel
+BUILD_PANEL=false
+
+# Default PHP Version
+PHP_VERSION="8.0"
```

Comment:
Can the current release run on 8.1? Because if it can we can just convert everything to 8.1 and not install different php versions based on version of panel.

Reply:
No, currently ptero `v1.7.0` supports php 7.4 or 8.0.
Starting with V2 php will be 8.1 or later.